RussianFilter/sections/general_extensions.txt

```diff
@@ -897,6 +897,8 @@ sinoptik.com.ru#$#iframe[bb] { visibility: hidden!important; }
 arendator.ru#$#.arcticmodal-container { display: none!important; }
 arendator.ru#$#.arcticmodal-overlay { display: none!important; }
 arendator.ru#$#body { overflow: visible!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/31556
+englex.ru#$#body { overflow: visible!important; }
```

Comment:
it seems it is not required.
In any case - if `overflow` must be chanded, all other popup elements must be hidden using CSS.